### ISSUE TITLE
backup of settings and list of blocked applications

### ISSUE THREAD

**Issue by Wjxfi** [StarGW-net/karma-firewall#8]
If I take too long to make this list and my phone breaks, I'm gonna be really upset.

**Comment by StarGW-net** [comment]
We wouldn't want you to be upset!

Personally when I get  new phone every 3-4 years I clean up the apps I have installed and start again.

I'll have a look at an export config option in the future

**Comment by StarGW-net** [comment]
Ah I have the backup flag set in AndroidMainfest.ini

android:allowBackup="true"

This means the Google App data backup and restore should work:

https://support.google.com/android/answer/2819582?hl=en

Not quite as good in-app backup and restore, but should work for now.

**Comment by Quantum-Future** [comment]
Yes please, make en export and import option for backup and restore of allowed/blocked applications and configuration options.
